src/pages/MyPage.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from "@emotion/styled";
+import UserContext from "@src/contexts/UserContext";
+import { PATH } from "@src/router/Router";
+import theme from "@src/styles/kakaoTheme";
+import { useContext, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+
+function MyPage() {
+  const navigate = useNavigate();
+  const userContext = useContext(UserContext);
+
+  const RequireLogin = (path: string) => {
```

Comment:
아마 실수로 PascalCase로 쓰신 거겠죠..?
그리고 requireLogin보다는 이 함수의 목적(로그인 페이지로 리다이렉트)을 명시하는 쪽이 좋을 것 같아요.
그래서 저라면 아래처럼 쓸 것 같습니다!
```suggestion
  const redirectLogin = (path: string) => {
```